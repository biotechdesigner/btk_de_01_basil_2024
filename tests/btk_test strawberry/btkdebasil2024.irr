************************************************************************
pyfao56: FAO-56 Evapotranspiration in Python
Irrigation Data
Timestamp: 03/18/2024 11:17:59
************************************************************************
Comments: 2024 basil
************************************************************************
Year-DOY  Depth     fw
2024-077  33.00   0.30