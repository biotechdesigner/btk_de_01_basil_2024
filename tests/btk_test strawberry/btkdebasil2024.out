************************************************************************
pyfao56: FAO-56 Evapotranspiration in Python
Output Data
Timestamp: 03/18/2024 11:17:59
Simulation start date: 03/17/2024
Simulation end date: 03/17/2024
Soil method: D - Default FAO-56 homogenous soil bucket approach
************************************************************************
Comments: 2024 basil
************************************************************************
Year-DOY  Year  DOY  DOW      Date  ETref  tKcb   Kcb     h Kcmax    fc    fw   few      De    Kr    Ke      E     DPe    Kc    ETc     TAW TAWrmax    TAWb    Zr     p     RAW    Ks Kcadj ETcadj      T      DP    Dinc      Dr     fDr   Drmax  fDrmax      Db     fDb   Irrig    Rain  Year  DOY  DOW      Date
2024-077  2024  077  Sun  03/17/24  1.588 0.400 0.400 0.100 1.180 0.000 0.300 0.300   0.000 0.000 0.000  0.000  97.427 0.400  0.635  20.000 -99.999 -99.999 0.250 0.575  11.492 1.000 0.400  0.635  0.635  22.365 -99.999   0.000   0.000 -99.999 -99.999 -99.999 -99.999  33.000   0.000  2024  077  Sun  03/17/24